### PR TITLE
Update Docker/Devilbox install options

### PR DESCRIPTION
**Pull request by fkaminski**
As described on slack a few updates regarding Docker installation for Windows users

### CODE REVIEW

File: docs/manual/guides/local-installation/devilbox.de.md [hunk 1]
```diff
@@ -11,14 +11,10 @@ tags:
...
-{{% notice note %}}
-Um die Devilbox nutzen zu können muss _Docker_ und _Docker Compose_ auf deinem System installiert sein. Falls das noch
-nicht der Fall ist, kannst du dir die 
-[Devilbox Prerequisites Dokumentation](https://devilbox.readthedocs.io/en/latest/getting-started/prerequisites.html) für 
-mehr Informationen zur Installation dieser Programme durchlesen.
-{{% /notice %}}
+Installationen der aktuellen »[Docker Desktop](https://www.docker.com/products/docker-desktop)« Version stehen für 
+verschiedene Betriebssysteme, u. a. auch für z. B. 
+»[Windows 10 Home](https://docs.docker.com/docker-for-windows/install-windows-home/)« oder 
+»[Windows 10 Pro](https://docs.docker.com/docker-for-windows/install/)«, bereit.
```
Comment (fritzmg) [inline docs/manual/guides/local-installation/devilbox.de.md]: This removes the notice about having to install Docker and Docker Compose and only mentions Windows.

**Review by fritzmg — CHANGES_REQUESTED** [review]

**Comment by fkaminski** [comment]
OK, i have now deposited the appropriate links in the original note ...

**Comment by fkaminski** [comment]
Little more detail regarding Windows 10 and WSL 2

**Comment by fkaminski** [comment]
After reviewing the PR is not really necessary.
Even the hints about uid/gid (with WSL 2) - These are not necessary - but also do not hurt ...
So i close this ...
Thanks

**Pull request closed without merging**